pkg/router/router.go

```diff
@@ -23,6 +24,7 @@ type Router struct {
 	Subscriptions       SubscriptionConfig            // global subscription config
 	SubscriptionConfigs map[string]SubscriptionConfig // subscription config per subscription field
 	MaxDepth            int                           // allow limiting max depth of GraphQL recursion
+	RequestTimeout      time.Duration
```

Comment:
```suggestion
	RequestTimeout      *time.Duration
```

This value should make it possible to distinguish between defined as 0 and undefined states

Reply:
The reason why it's necessary to distinguish those two states is because the current undefined value should have the same semantics as the current behavior. And those differ from semantics of `0` value.